### ISSUE TITLE
二分类问题 - sigmoid - softmax

### ISSUE THREAD

**Issue by junxnone** [junxnone/AI#50]
**常识中：**
- sigmoid 用于二分类问题
> 最后一层使用 sigmoid 激活以输出一个 0∼1 范围内的概率值（表示样本的目标值等于 1 的可能性，即评论为正面的可能性）
- softmax 用于多分类问题
> 最后一层使用 softmax 激活以输出在 classes_num 个不同输出类别上的概率分布

# Reference
- [ ] [Keras - Softmax 与 Sigmoid 二分类问题分析](https://www.aiuai.cn/aifarm679.html)
- [ ] [Keras深度学习笔记(三)：神经网络入门](https://xiaosheng.me/2018/11/24/article154/)
- [ ] [sigmoid_and_softmax.ipynb](https://gist.github.com/ypwhs/6905ebbda99d04621f9fc00417657ae2)

# **二分类时** sigmoid 和softmax 等价
- **sigmoid**
![image](https://user-images.githubusercontent.com/2216970/69112835-4078a800-0abc-11ea-81e4-6e7b0c1833b9.png)
- **softmax**
![image](https://user-images.githubusercontent.com/2216970/69112891-669e4800-0abc-11ea-9068-25724edc9e15.png)
> 当 ![image](https://user-images.githubusercontent.com/2216970/69113402-dcef7a00-0abd-11ea-91bb-343c52144deb.png) 时：
> ![image](https://user-images.githubusercontent.com/2216970/69113422-ea0c6900-0abd-11ea-9090-4e2e28364478.png)

# 对比
- Sigmoid 与 Softmax 完全等价
- Sigmoid 与 Softmax 分类器的权值可以相互转换
- Softmax 会比 Sigmoid 浪费 2 倍的权值空间
